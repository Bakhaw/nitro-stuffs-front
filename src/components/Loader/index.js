import React from 'react';
import PropTypes from 'prop-types';

import CircularProgress from '@material-ui/core/CircularProgress';

function Loader({ size = 30 }) {
  return (
    <div>
      <CircularProgress size={size} />
    </div>
  );
}

Loader.propTypes = {
  size: PropTypes.number
};

export default Loader;
