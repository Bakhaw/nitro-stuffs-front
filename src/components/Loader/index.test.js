import React from 'react';
import { shallow } from 'enzyme';

import Loader from '.';

test('should render properly', () => {
  const container = shallow(<Loader />);
  expect(container.debug()).toMatchSnapshot();
});
