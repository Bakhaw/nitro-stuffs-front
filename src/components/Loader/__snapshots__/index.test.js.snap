// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
"<div>
  <WithStyles(ForwardRef(CircularProgress)) size={30} />
</div>"
`;
